core/arch/arm/mm/core_mmu_v7.c

```diff
@@ -69,9 +69,16 @@
 /* Normal memory, Outer Write-Back Write-Allocate Cacheable */
 #define TEE_MMU_TTB_RNG_WBWA    (1 << 3)
 
+/* Normal memory, Outer Write-Back no Write-Allocate Cacheable */
+#define TEE_MMU_TTB_RNG_WB      (3 << 3)
+
+#ifndef CFG_NO_SMP
```

Comment:
It is valid to have one core in SMP, we do this on some of our platforms for coherency reasons with non-ARM cores.
TTBCR should be fine, and this is only a fix for ARMv7.